releases.go

```diff
@@ -315,6 +321,17 @@ func newSchedulerProcess(release *Release, name string, p Process) *scheduler.Pr
 		"empire.app.process": name,
 	}
 
+	var exposure *scheduler.Exposure
+	// For `web` processes defined in the standard procfile, we'll
+	// generate a default exposure setting and also set the PORT
+	// environment variable for backwards compatability.
+	if name == webProcessType && len(p.Ports) == 0 {
+		exposure = standardWebExposure(release.App)
+		env["PORT"] = "8080"
+	} else {
+		exposure = processExposure(release.App, p)
```

Comment:
Right. For backwards compatibility, if you defined a `web` proc, but no `ports:` then it's equivalent to:

```yaml
web:
  environment:
    PORT: "8080"
  ports:
    - "80:8080":
        protocol: "http"
    - "443:8080": # If a cert is provided
        protocol: "https"
```